two_factor/templates/two_factor/core/login.html

```diff
@@ -31,7 +31,7 @@
     {% include "two_factor/_wizard_forms.html" %}
 
     {# hidden submit button to enable [enter] key #}
-    <input type="submit" value="" class="d-none" />
+    <input hidden type="submit" value="" />
```

Comment:
I've just notices that you change styling of templates in the /core/ folder. They should not be used in django admin, so it's ok to leave them with bootstrap classes

Reply:
I removed it for the core/login.html file but not in the others file as they are present threw the admin login process after the the user logged in